doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -219,6 +219,71 @@ and ``coqtop``, unless stated otherwise:
 :-batch: Exit just after argument parsing. Available for ``coqtop`` only.
 :-verbose: Output the content of the input file as it is compiled.
   This option is available for ``coqc`` only.
+:-native-compiler (yes|no|ondemand): Enable the :tacn:`native_compute`
+  reduction machine and precompilation to ``.cmxs`` files for future use
+  by :tacn:`native_compute`.
+  Setting ``yes`` enables :tacn:`native_compute`; it also causes Coq
+  to precompile the native code for future use; all dependencies need
+  to have been precompiled beforehand. Setting ``no`` disables
+  :tacn:`native_compute` which defaults back to :tacn:`vm_compute`; no files are precompiled.
+  Setting ``ondemand`` enables :tacn:`native_compute`
+  but disables precompilation; all missing dependencies will be recompiled
+  every time :tacn:`native_compute` is called.
```

Comment:
- Some readers might not realize this "recompilation" has no caching.
- Does "every time" mean that `native_compute. native_compute.` would recompile dependencies twice?
If not, I'd recommend a different text — "missing dependencies will be recompiled as needed for files :tacn:`native_compute` is called`. I thought they'd be cached within the same file, but I'd rather not guess.

Reply:
- Skimming the review comments, I think the "temporary" recompilation went in the right direction...
- Being too specific on `ondemand` might not be worth it.